prototype/doctors-ui/src/App.css

```diff
@@ -31,11 +31,11 @@ html,
 }
 
 .preview {
-  max-width: 400px;
+  max-height: 600px;
 }
 
 .preview-container {
-  border: 2px dashed #AAAa;
-  height: 400px;
-  width: 400px;
+  /* border: 2px dashed #AAAa; */
+  /* width: 400px;
+  height: 400px; */
```

Comment:
i think both preview and file-upload are using these classNames, it would make sense to rename the classNames to something better in the file-upload one. e.g. fileUploadContainer and fileUpload, that way we can preserve that style in the fileupload